cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -146,7 +153,7 @@ void run_mnist(const std::string& model_name,
     std::cout<<"All ranks have Resnet image"<<std::endl;
 
   double loop_start = MPI_Wtime();
-  for (int i = 0; i < 101; i++) {
+  for (int i = 0; i < iterations; i++) {
```

Comment:
There is one extra iteration done to warm up caches, so this should actually be `iterations + 1`